lib/huginn_acumen_order_agent/concerns/invoice_detail_query_concern.rb

```diff
@@ -41,9 +41,10 @@ def process_invoice_detail_response(raw_data)
         results << mapped_item
       rescue => error
         issue_error(AcumenOrderError.new(
+          error[:status],
           'process_invoice_detail_response',
           'Failed while processing Invoice_Detail record',
-          invoice_details,
+          { invoice_details_id: invoice_details[:id] },
```

Comment:
invoice_details is not a hash here. It's XML data

Reply:
@mrogers-5s Made some more updates here let me know what you think.